### DISCUSSION TITLE
Quick question,where is the data*or history stored?I want to backup them or change the storage location

### DISCUSSION THREAD

**Discussion by RESDXChgfore9hing** [ActivityWatch/activitywatch#820] [Q&A] [answered]
- [ - ] I have searched the [issues](https://github.com/ActivityWatch/activitywatch/issues) of this repo and believe that this is not a duplicate.
- [  ?] I have searched the [documentation](https://docs.activitywatch.net/en/latest/) and believe that my question is not covered.

## Issue

 I wanted to back the data and maybe even make it portable*.Huge thanks.
Extremely sorry if its written in somewhere.

**Comment by BelKed** [comment, marked as answer]
It's written in the FAQ section of the [documentation](https://docs.activitywatch.net/en/latest/)...

> Where is my data stored?
> -------------------------
>
> All your data is stored in a SQLite database in the [data directory](https://docs.activitywatch.net/en/latest/directories.html#data-directory).

<sub>(https://docs.activitywatch.net/en/latest/faq.html#where-is-my-data-stored)</sub>

**Reply by RESDXChgfore9hing** [reply]
Thanks,btw is it possible to change the data directory?I thing its not in the settings but if its possible then it would be cool.

**Reply by ErikBjare** [reply]
@RESDXChgfore9hing As the [docs on directories](https://docs.activitywatch.net/en/latest/directories.html) states there is a `XDG_CONFIG_HOME` environment variable for *config files*. (according to the [XDG basedir spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html))

Similarly, there should be a `XDG_DATA_HOME` for *data files*, but seems to be undocumented. (If it doesn't work, let us know!)

These environment variables are probably only respected on Linux, however.

**Reply by RESDXChgfore9hing** [reply]
oh thats sad,it would be great if its cross platform but that you very much.

**Reply by solucionesuno** [reply]
so is no way to change storage folder for windows?
